### PR TITLE
add cookie jar to http request

### PR DESCRIPTION
**Pull request by MaximeHeckel**

### CODE REVIEW

**Comment by fermayo** [comment]
@tifayuki please review

File: dockercloud/http.go [hunk 1]
```diff
@@ -5,10 +5,13 @@ import (
 	"fmt"
 	"io/ioutil"
 	"net/http"
+	"net/http/cookiejar"
 	"os"
 )
 
 var customUserAgent = "go-dockercloud/" + version
+var client *http.Client
+var jar http.CookieJar
```
Comment (tifayuki) [inline dockercloud/http.go]: It is unnecessary to declare jar as a global variable, right?

Comment (MaximeHeckel) [inline dockercloud/http.go]: yep, I pushed again and put the exact same implementation as the event container.

**Comment by tifayuki** [comment]
LGTM

**Comment by MaximeHeckel** [comment]
Cool, once merged I'll recompile and push network-daemon

**Pull request merged**